decidim-proposals/app/views/decidim/proposals/proposals/show.html.erb

```diff
@@ -17,7 +17,7 @@
         <%= link_to_current_or_new_conversation_with(@proposal.author) %>
       <% end %>
     </div>
-    <% unless @proposal.withdrawn? %>
+    <% unless @proposal.withdrawn? or !@proposal.withdrawable_by?(current_user) %>
```

Comment:
The `unless...or` confuses me a lot, I think it's easier to read this way:

`if !@proposal.withdrawn? && @proposal.withdrawable_by?(current_user)`

And if you add the `withdrawn?` check to `withdrawable_by` then it's just:

`if withdrawable_by?(current_user)`

Reply:
I agree with you!

Code refactorization done! ☺️